pkg/k8shandler/util.go

```diff
@@ -126,9 +128,93 @@ func isValidConf(dpl *api.Elasticsearch) error {
 		}
 	}
 
+	if ok, msg := hasValidUUIDs(dpl); !ok {
+		if err := updateInvalidUUIDChangeCondition(dpl, v1.ConditionTrue, msg); err != nil {
+			return err
+		}
+		return fmt.Errorf("Unsupported change to UUIDs made: %v", msg)
+	} else {
+		if err := updateInvalidUUIDChangeCondition(dpl, v1.ConditionFalse, ""); err != nil {
+			return err
+		}
+	}
+
 	return nil
 }
 
+func hasValidUUIDs(dpl *api.Elasticsearch) (bool, string) {
+
+	// TODO:
+	// check that someone didn't update a uuid
```

Comment:
Maybe a good reason to prefer some hash of name?

Reply:
the UUID is what is making the name random...